templates/kops/kops-private-topology.yaml.gotmpl

```diff
@@ -44,6 +44,12 @@ spec:
     anonymousAuth: false
     authorizationMode: {{ getenv "KOPS_KUBE_API_SERVER_AUTHORIZATION_MODE" "RBAC" }}
   {{- end }}
+  {{- if bool (getenv "KOPS_KUBE_API_SERVER_FEATURE_GATES_ENABLED" "false") }}
```

Comment:
Is this not redundant with `KOPS_KUBE_API_SERVER_ENABLED_FEATURES`? What if we only enable if `KOPS_KUBE_API_SERVER_ENABLED_FEATURES` is non-empty

Reply:
I can certainly do that.  I think that's cleaner.